### DISCUSSION TITLE
关于在overleaf中使用的问题

### DISCUSSION THREAD

**Discussion by LouSQ17** [Aetf/xjtuthesis#51] [Q&A]
作者你好，我在overleaf中使用本模板。24年毕设要求是正文字体为小四，但是我在overleaf中编译得到的字体大小却为为10pt，修改.cls文件中的/normalize后的文字大小也没有变化，请问这是overleaf的问题还是我修改方式不正确呢。又应该如何使其在overleaf中正常显示呢？

**Comment by LouSQ17** [comment]
更正一下，.cls文件中修改的是/normalsize

**Comment by LouSQ17** [comment]
再补充一下细节，如果不在bachelor中使用ctex的话，修改normalsize是有效的，但是此时中文是乱码，如果再bachelor中使用ctex，则修改normalsize无效

**Comment by LouSQ17** [comment]
目前初步的一个解决办法，tex文件中不使用ctex包，打开propfont，单独下载一个SimSun.ttf，把.cls文件中的SimSun改为SimSun.ttf，这样中文可以正常显示，并且可以调节大小。

**Comment by Zipper-1** [comment]
Overleaf由于版权问题无法支持宋体(SimSun)。可以通过手动添加的方式保证字体格式生效，或是在Windows上使用textlive等latex发行版进行生成。
